### DISCUSSION TITLE
LaraDumps For JS Environment

### DISCUSSION THREAD

**Discussion by devduttabain** [laradumps/laradumps#143] [Ideas]
# 🛻 LaraDumps Feature Request

## Summary

Extend LaraDumps functionality to support JavaScript/Node.js development environments like Node.js, Vue, React, etc., similar to the Spatie Ray tool
### Why is this needed?

LaraDumps is a great tool for debugging and inspecting data in Laravel applications. However, as the web development landscape evolves, more and more applications are built using JavaScript frameworks and libraries like Vue, React, and Node.js. Developers working with these technologies would greatly benefit from having a tool similar to LaraDumps that allows them to easily inspect and debug their code during development.

**Comment by luanfreitasdev** [comment]
Hi!, thanks for the suggestion.

I want this functionality in laradumps natively, but now I cannot dedicate myself to it. But if you have any suggestions on how to do it or if you want to create a package to support it, I also don't see a problem.

Thanks

**Comment by devduttabain** [comment]
Thank you for your response.
While I cannot dedicate focused effort towards developing a package for this at the moment, I may consider starting a separate package tailored for JavaScript environments if an opportunity arises. I would be delighted to discuss any ideas or suggestions you might have. LaraDumps is a fantastic tool that has greatly enhanced my productivity, and I sincerely appreciate your dedication to this project.

**Comment by luanfreitasdev** [comment]
I would love to see a simple script that connects to the hard-coded app and sends just one post to the app. Then we can improve it little by little. It doesn't need to be as complex as Ray 😄

**Reply by devduttabain** [reply]
On it 😊!

**Comment by anburocky3** [comment]
Idea: Can we integrate Inerta support in this?
